html-web-page/index.html

```diff
@@ -0,0 +1,96 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <meta http-equiv="X-UA-Compatible" content="ie=edge">
+  <title>HTML Web Page</title>
+  <style>
+    table,
+    th,
+    td {
+      border: 1px solid black;
+    }
+
+    td:not(:first-child) {
+      text-align: center;
+    }
+  </style>
+</head>
+<body>
+
+  <!-- your HTML code here 👇 -->
+  <div>
+    <h1>Jelly of the Month Club</h1>
+  </div>
+
+  <div>
+    <h2>Make all your friends jelly of your stock of jelly!</h2>
+  </div>
+  <div>
+    <p> Here at JOTMC Inc., we are constantly striving to supply our customers with the highest-quality
+      and most unique jellies on a monthly basis. For the Jelly love in your life (or yourself!),
+      this is the gift that keeps on giving all year round. We offer three tiers of monthly jelly joy, and each
+      one offeres their own perks. Here is a breakdown of each, as well as your monthly price: </p>
+  </div>
+  <div id="jelly table">
+    <table>
+      <thead>
+        <tr>
+          <th> </th>
+          <th>Basic Plan</th>
+          <th>Deluxe Plan</th>
+          <th>Jelly Giant Plan</th>
+        </tr>
+      </thead>
+      <tbody>
+        <tr>
+          <td>Three new curated jellies per month</td>
+          <td>&check;</td>
+          <td>&check;</td>
+          <td>&check;</td>
+        </tr>
+        <tr>
+          <td>Monthly jellies come with commemorative mason jars</td>
+          <td></td>
+          <td>&check;</td>
+          <td>&check;</td>
+        </tr>
+        <tr>
+          <td>Free bonus jar of preserves bi-monthly</td>
+          <td></td>
+          <td>&check;</td>
+          <td>&check;</td>
+        </tr>
+        <tr>
+          <td>Free yearly tickets to Jelly Con</td>
+          <td></td>
+          <td></td>
+          <td>&check;</td>
+        </tr>
+        <tr>
+          <td>Montly Pricing</td>
+          <td>$30</td>
+          <td>$50</td>
+          <td>$60</td>
+        </tr>
+      </tbody>
+    </table>
+  </div>
+  <div>
+    <p>Like what you see? Become a part of the JOTMC Family today!</p>
+  </div>
+  <div>
+    <form>
```

Comment:
`form` elements are already block-level elements, so the surrounding `div` element is unnecessary here.